lib/sdf-server/src/service/session.rs

```diff
@@ -73,10 +69,12 @@ pub type SessionResult<T> = std::result::Result<T, SessionError>;
 impl IntoResponse for SessionError {
     fn into_response(self) -> Response {
         let (status_code, error_message) = match self {
-            SessionError::LoginFailed => (StatusCode::CONFLICT, self.to_string()),
-            SessionError::InvalidWorkspace(_) => (StatusCode::CONFLICT, self.to_string()),
-            SessionError::WorkspacePermission(_) => (StatusCode::UNAUTHORIZED, self.to_string()),
-            SessionError::AuthApiError(_) => (StatusCode::UNAUTHORIZED, self.to_string()),
+            Self::LoginFailed => (StatusCode::CONFLICT, self.to_string()),
+            Self::Workspace(WorkspaceError::WorkspaceNotFound(_)) => {
```

Comment:
Is a StatusCode::CONFLICT what we want here?

Reply:
NOT_FOUND sounds better to me.

Reply:
i think that makes more sense!

Reply:
Especially since this is for session endpoints, I'm worried we have callers that are sensitive to the error codes; unless you feel strongly, I don't think this is a good PR to change this behavior in.